README.md

```diff
@@ -58,6 +58,30 @@ try {
 }
 ````
 
+If the values from `EvaluationResponse` class doesn't matter and only a Color is needed:
+````java
+flagr = new Flagr(FLAGR_HOST);
+Color color = flagr.evaluateAndGetVariantAttachment(
+        new EvaluationContext("color"),
+        Color.class,
+        new Color("white", "#FFFFFF") // default value returned when the flag is disabled or there's some connection problem.
+);
+````
+
+When only the variant matters, use:
+````java
+flagr = new Flagr(FLAGR_HOST);
+String variant = flagr.evaluateVaariant(new EvaluationContext("myflag")));
+````
+
+And if it's just a simple on/off flag. Use the `evaluateBoolean` method:
```

Comment:
Maybe extend the description here to say something about what variant keys correspond to a `true` evaluation.

Reply:
👍